yamale/yamale.py

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env python
 import sys
+import os
 from .schema import Schema
 
 PY2 = sys.version_info[0] == 2
 
 
 def make_schema(path, parser='PyYAML', validators=None):
     # validators = None means use default.
+    # test if the schema.yaml file is empty
```

Comment:
I've been thinking a little about this. How about checking `raw_schemas` instead of the size of the file. For example:
```python
    if not raw_schemas:
        raise ValueError('{} is an empty file!'.format(path))
```

Or you can even place it within the current try block for something like this:
```python
    try:
        if not raw_schemas:
            raise ValueError('No schema documents found!')
        s = Schema(raw_schemas[0], path, validators=validators)
        # Additional documents contain Includes.
        for raw_schema in raw_schemas[1:]:
            s.add_include(raw_schema)
    except (TypeError, SyntaxError, ValueError) as e:
        error = 'Schema error in file %s\n' % path
        error += str(e)
        if PY2:
            error.encode('utf-8')
        raise SyntaxError(error)
```

Reply:
I choose first one, so that tests I added doesn't require any further manipulation; lemme know otherwise.